buildpipeline/Core-Setup-CentOS-x64.json

```diff
@@ -124,7 +124,7 @@
       "allowOverride": true
     },
     "BuildArguments": {
-      "value": "--skip-prereqs --configuration $(BuildConfiguration)  --docker centos.7 --targets Default",
+      "value": "--skip-prereqs --configuration $(BuildConfiguration)  --docker centos.7 --targets Default --env-vars 'NUGET_API_KEY=$(NUGET_API_KEY)', 'GITHUB_PASSWORD=$(GITHUB_PASSWORD)', 'REPO_PASS=$(REPO_PASS)'",
```

Comment:
Does this formatting work with the spaces in between?  I don't think the parser will pick `GITHUB_PASSWORD` since there is a space before it.

Reply:
I also think the `'` characters are unnecessary, and may not work.

---
In reply to: [173847910](https://github.com/dotnet/core-setup/pull/3811#discussion_r173847910) [](ancestors = 173847910)

Reply:
@eerhardt what do you mean by 

> Does this formatting work with the spaces in between?

Are you referring to the space between the comma after `NUGET_API_KEY` and the single quote before `GITHUB_PASSWORD`?

Reply:
Yes:

`'NUGET_API_KEY=$(NUGET_API_KEY)', 'GITHUB_PASSWORD=$(GITHUB_PASSWORD)'`

I don't think spaces in between is going to work.